CRM/CiviAwards/Form/AwardReview.php

```diff
@@ -125,6 +125,9 @@ public function preProcess() {
       $this->caseId = CRM_Utils_Array::first($this->activity['case_id']);
     }
 
+    if ($this->isCaseApplicationDeleted()) {
+      drupal_not_found();
```

Comment:
Since Civicase is a civicrm extension and it's possible that the extension can be used with Civicrm in combination with another CMS like wordpress or Joomla. We should not have drupal specific code here. 

I would suggest we move the logic to check deleted applications to the getErrorMessage function and return the appropriate error message there.  See : https://github.com/compucorp/uk.co.compucorp.civiawards/blob/DRU-190-deleted-applications-access/CRM/CiviAwards/Form/AwardReview.php#L164-L189

This will also be consistent with previous error message display implementation.

Reply:
Also we need to check if we want this behaviour on the Civicrm side. If not, we can add the `$this->isReviewFromSsp()` in addition to the application deleted condition if we need to restrict to SSP only.

Reply:
fixed